rules/windows/process_creation/win_netsh_fw_add.yml

```diff
@@ -17,11 +17,11 @@ logsource:
     product: windows
 detection:
     selection1:
-        CommandLine:
-            - '*netsh*'
+        CommandLine|contains:
+            - 'netsh'
     selection2:
-        CommandLine:
-            - '*firewall add*'
+        CommandLine|contains:
+            - 'firewall add'
```

Comment:
could you please change it to the following:

```
        CommandLine|contains|all:
            - 'firewall'
            - 'add'
```

Reply:
Applied